ichnaea/data/tests/test_public.py

```diff
@@ -201,8 +201,8 @@ def test_new_stations(self, session, redis_client, cellarea_queue):
         assert wcdma.min_lon == wcdma.lon
         assert wcdma.radius == 0
         assert wcdma.samples == 6
-        assert wcdma.created == datetime(2019, 9, 11, 16, 49, 24, tzinfo=UTC)
-        assert wcdma.modified == datetime(2019, 10, 3, 16, 31, 56, tzinfo=UTC)
+        assert wcdma.created == datetime(2019, 9, 11, 16, 49, 24, tzinfo=ZoneInfo("UTC"))
+        assert wcdma.modified == datetime(2019, 10, 3, 16, 31, 56, tzinfo=ZoneInfo("UTC"))
```

Comment:
The linter wants to break up these lines because they are now too long:

```suggestion
        assert wcdma.created == datetime(
            2019, 9, 11, 16, 49, 24, tzinfo=ZoneInfo("UTC"))
        assert wcdma.modified == datetime(
            2019, 10, 3, 16, 31, 56, tzinfo=ZoneInfo("UTC"))
```